.ai/spec/how/fbc-release.md

```
@@ -0,0 +1,78 @@
+For OCP 5.x (v2 bundle), analogous files are created in the same directory:
+
+```
+  release-plan-fbc-v2-prod.yaml     # prod ReleasePlan per OCP 5.x version (v2 bundle)
+  release-plan-fbc-v2-staging.yaml  # staging ReleasePlan per OCP 5.x version (v2 bundle)
+  release-plan-bundle-v2.yaml       # bundle-level release (v2)
+```
+
+## Adding a New OCP 4.x Version (e.g., 4.23)
+
+1. **Extend the v1 bundle annotation** in `bundle/metadata/annotations.yaml`:
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)